testsuite/MDAnalysisTests/__init__.py

```diff
@@ -114,6 +114,9 @@
 """
 from __future__ import absolute_import
 import logging
+
+import pytest
+
```

Comment:
@kain88-de Do I have to update all the comments involving nose in all the files?